build.sbt

```diff
@@ -867,37 +930,36 @@ lazy val `scio-extra`: Project = project
   .settings(
     description := "Scio extra utilities",
     libraryDependencies ++= Seq(
-      "com.chuusai" %% "shapeless" % shapelessVersion,
       "com.google.apis" % "google-api-services-bigquery" % googleApiServicesBigQueryVersion,
+      "com.google.protobuf" % "protobuf-java" % protobufVersion,
+      "com.google.zetasketch" % "zetasketch" % zetasketchVersion,
+      "com.nrinaudo" %% "kantan.codecs" % kantanCodecsVersion,
       "com.nrinaudo" %% "kantan.csv" % kantanCsvVersion,
       "com.spotify" % "annoy" % annoyVersion,
       "com.spotify.sparkey" % "sparkey" % sparkeyVersion,
       "com.twitter" %% "algebird-core" % algebirdVersion,
-      "info.debatty" % "java-lsh" % javaLshVersion,
       "io.circe" %% "circe-core" % circeVersion,
       "io.circe" %% "circe-generic" % circeVersion,
       "io.circe" %% "circe-parser" % circeVersion,
       "joda-time" % "joda-time" % jodaTimeVersion,
+      "net.java.dev.jna" % "jna" % jnaVersion, // used by annoy4s
```

Comment:
@clairemcginty or @shnapz this should fix issues with M1.
the plugin required me to add the lib explicitly